src/migrations/2015_11_23_311097_create_tracker_language_table.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+use PragmaRX\Tracker\Support\Migration;
+
+class CreateTrackerLanguagesTable extends Migration {
```

Comment:
Class is incorrectly named. Should be CreateTrackerLanguageTable.
But I fixed in my end.